### PR TITLE
Add UDF for url_encode / url_decode

### PR DESCRIPTION
**Pull request by 1987yama3**
I implement a BigQuery UDF to convert url encode and url decode.

Thanks

### CODE REVIEW

File: udfs/community/test_cases.js [hunk 1]
```diff
@@ -308,6 +308,66 @@ generate_udf_test("typeof", [
         expected_output: `"NUMERIC"`
     },
 ]);
+generate_udf_test("url_decode", [
```
Comment (danieldeleo) [inline udfs/community/test_cases.js:311]: Add test cases to both methods which pass NULL to the text input parameter

File: udfs/community/url_encode.sqlx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+config { hasOutput: true }
+/*
+ * Copyright 2021 Google LLC
```
Comment (danieldeleo) [inline udfs/community/url_encode.sqlx:3]:
```suggestion
 * Copyright 2022 Google LLC
```

**Comment by 1987yama3** [comment]
@danieldeleo Thanks for your review. I fix it.

**Review by danieldeleo — APPROVED** [review]
Thanks for contributing these UDFs!
